cc/algorithms/util.cc

```diff
@@ -43,6 +43,44 @@ double DefaultEpsilon() { return std::log(3); }
 
 double GetNextPowerOfTwo(double n) { return pow(2.0, ceil(log2(n))); }
 
+
+double inverseErrorFunction(double x) {
+
+  double LESS_THAN_FIVE_CONSTANTS[] = {2.81022636*pow(10,-8), 3.43273939*pow(10,-7), 
```

Comment:
Is there a way to directly define those without calling pow()?

Reply:
Yes! Thank you!